metadata-ingestion/src/datahub/ingestion/source/superset.py

```diff
@@ -419,3 +441,18 @@ def get_workunit_processors(self) -> List[Optional[MetadataWorkUnitProcessor]]:
 
     def get_report(self) -> StaleEntityRemovalSourceReport:
         return self.report
+
+    def _gen_domain_urn(self, title: str) -> Optional[str]:
```

Comment:
we have a "DomainRegistry" helper, which lets us convert domain names into urns using a lookup against DataHub - let's see if we can reuse the same pattern

A good example to copy would be the kafka integration's domain support: https://github.com/datahub-project/datahub/blob/1741c07d769f56a9cf066172725384b4e8780839/metadata-ingestion/src/datahub/ingestion/source/kafka.py#L445

Reply:
Ok thanks, it's done :+1: Let me know if it is what you expected

Reply:
Although, I removed the `_get_domain_urn` to have only one function `_get_domain_wu` (I don't think it's relevant to have two distinct functions here)